### ISSUE TITLE
Error 2 while installing Ruby 2.1.2

### ISSUE THREAD

**Issue by ryzokuken** [rvm/rvm#4106]
## Description

Whenever trying to install ruby 2.1.2 using `rvm install 2.1.2`, I get the following error:

```
Error running '__rvm_make -j4',
showing last 15 lines of /home/ryzokuken/.rvm/log/1500125001_ruby-2.1.2/make.log
make[2]: *** [Makefile:280: ossl_engine.o] Error 1
make[2]: *** [Makefile:280: ossl_cipher.o] Error 1
make[2]: Leaving directory '/home/ryzokuken/.rvm/src/ruby-2.1.2/ext/openssl'
make[1]: *** [exts.mk:187: ext/openssl/all] Error 2
make[1]: *** Waiting for unfinished jobs....
installing default callback libraries
linking shared-object pathname.so
make[2]: Leaving directory '/home/ryzokuken/.rvm/src/ruby-2.1.2/ext/pathname'
linking shared-object nkf.so
make[2]: Leaving directory '/home/ryzokuken/.rvm/src/ruby-2.1.2/ext/nkf'
linking shared-object dl/callback.so
make[2]: Leaving directory '/home/ryzokuken/.rvm/src/ruby-2.1.2/ext/dl/callback'
make[1]: Leaving directory '/home/ryzokuken/.rvm/src/ruby-2.1.2'
make: *** [uncommon.mk:180: build-ext] Error 2
++ return 2
There has been an error while running make. Halting the installation.
```

## Steps to reproduce

 1. ... Distro: Antergos, Session: GNOME Wayland
 2. ... RVM version: rvm 1.29.2 (master) by Michal Papis, Piotr Kuczynski, Wayne E. Seguin [https://rvm.io/]
 3. ... Command Run: rvm install 2.1.2

## Expected behavior

That ruby 2.1.2 is installed. It always used to work on my other installations.

## Actual behavior

* [command](https://gist.github.com/...)
* [config.log](https://gist.github.com/...) - only for issues with `install`
* [make.log](https://gist.github.com/...) - only for issues with `install`

**Debug mode**:
* [command](https://gist.github.com/...)
* [config.log](https://gist.github.com/...) - only for issues with `install`
* [make.log](https://gist.github.com/ryzokuken/4c43533495cfddcb814b7af4aa0f74d8) - only for issues with `install`

## Environment info

```
~/Code took 1m 20s 
➜ rvm info

system:

  system:
    uname:        "Linux black-perl 4.11.9-1-ARCH #1 SMP PREEMPT Wed Jul 5 18:23:08 CEST 2017 x86_64 GNU/Linux"
    name:         "Arch"
    version:      "libc-2.25"
    architecture: "x86_64"
    bash:         "/usr/bin/bash => GNU bash, version 4.4.12(1)-release (x86_64-unknown-linux-gnu)"
    zsh:          "/usr/bin/zsh => zsh 5.3.1 (x86_64-unknown-linux-gnu)"
    remote_path:  "arch/libc-2.25/x86_64"

  rvm:
    version:      "1.29.2 (master)"
    updated:      "10 minutes 5 seconds ago"
    path:         "/home/ryzokuken/.rvm"
    autolibs:     "[4] Allow RVM to use package manager if found, install missing dependencies, install package manager (only OS X)."

  homes:
    gem:          "not set"
    ruby:         "not set"

  binaries:
    ruby:         ""
    irb:          ""
    gem:          ""
    rake:         ""

  environment:
    PATH:         "/home/ryzokuken/.cargo/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl:/home/ryzokuken/.rvm/bin:/usr/local/go/bin:/home/ryzokuken/Golang/bin:/home/ryzokuken/.rvm/bin"
    GEM_HOME:     ""
    GEM_PATH:     ""
    MY_RUBY_HOME: ""
    IRBRC:        ""
    RUBYOPT:      ""
    gemset:       ""

```

**Comment by mpapis** [comment]
Most likely will be fixed with #4033 - closing.

**Comment by Alex9532** [comment]
It's still happening. I'm on macOS 10.15.7, and I'm getting the exact same error, but on Ruby 3.0.0. Help
